### PR TITLE
Address warning

### PR DESCRIPTION
**Pull request by Soptq**
Address the warning of using the last argument as keyword parameters

#7

### CODE REVIEW

File: include_absolute.rb [hunk 1]
```diff
@@ -147,7 +147,7 @@ def realpath_prefixed_with?(path, dir)
 
       # This method allows to modify the file content by inheriting from the class.
       def read_file(file, context)
-        File.read(file, file_read_opts(context))
+        File.read(file, **file_read_opts(context))
```
Comment (tnhu) [inline include_absolute.rb:150]: @Soptq Thanks for the PR. Will this change break under Ruby 2?

Comment (Soptq) [reply include_absolute.rb:150]: I don’t think so, it is a official code updating method mentioned at https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/

Comment (Soptq) [reply include_absolute.rb:150]: I just make a quick Jekyll 4.1 site build under Ruby 2.7 and it works perfectly. In terms of lower version, I think they are very likely to work as well, but I can’t confirm it as Jekyll 4.1 will raise exception if Ruby is under lower than 2.7 version.

Comment (tnhu) [reply include_absolute.rb:150]: Thanks for confirmation, I'm merging it.

**Pull request merged**